src/_gettsim/parameters/ges_rente.yaml

```diff
@@ -1836,42 +2199,36 @@ altersgrenze_besond_langj_versicherte:
     en: null
   type: piecewise_linear
   2012-01-01:
```

Comment:
Should this be the date of announcement or the date when this came into effect? I think we already discussed this in #645 and went for date of announcement (because difference didn't matter since affected cohorts were sufficiently far away from retirement age). Here, the difference matters, because the reform applies to all cohorts and was announced 5 years before it came into effect (2007 vs 2012).

Imo in this case we should go for the date when it came into effect, but then I'm worried regarding inconsistencies concerning how we set the other dates in this parameter file.

Reply:
Announcement is fine, won't ever be used before 2012 due to the setting of dates_active. But mention in the notes here so people are not confused.

Reply:
That's true, we could just let the function and parameter dates differ. Didn't think about that.